src/Haskoin/Address.hs

```diff
@@ -5,16 +5,15 @@
 {-# LANGUAGE LambdaCase #-}
 {-# LANGUAGE OverloadedStrings #-}
 
-{- |
-Module      : Haskoin.Address
-Copyright   : No rights reserved
-License     : MIT
-Maintainer  : jprupp@protonmail.ch
-Stability   : experimental
-Portability : POSIX
-
-Base58, CashAddr, Bech32 address and WIF private key serialization support.
--}
+-- |
+--Module      : Haskoin.Address
+--Copyright   : No rights reserved
+--License     : MIT
+--Maintainer  : jprupp@protonmail.ch
+--Stability   : experimental
+--Portability : POSIX
+--
+--Base58, CashAddr, Bech32 address and WIF private key serialization support.
```

Comment:
No `CashAddr` (and below, prob just grep it)

Reply:
Good catch.